fix compile Problems if you VDR compiled with bigpatch

Joerg Bornkessel <hd_brummy@gentoo.org> (06 May 2006)

diff -Naur vdr-ffnetdev-0.1.0.orig/ffnetdev.c vdr-ffnetdev-0.1.0/ffnetdev.c
--- vdr-ffnetdev-0.1.0.orig/ffnetdev.c	2006-05-06 21:00:39.433264204 +0000
+++ vdr-ffnetdev-0.1.0/ffnetdev.c	2006-05-06 21:02:31.248100382 +0000
@@ -127,8 +127,12 @@
   return true;
 }
 
-bool cPluginFFNetDev::Active(void) {
-	return (cOSDWorker::Active() || cTSWorker::Active());
+cString cPluginFFNetDev::Active(void) {
+	if(cOSDWorker::Active() || cTSWorker::Active())
+		{
+			return NULL;
+		}
+	return NULL;
 }
 
 bool cPluginFFNetDev::Start(void)
diff -Naur vdr-ffnetdev-0.1.0.orig/ffnetdev.h vdr-ffnetdev-0.1.0/ffnetdev.h
--- vdr-ffnetdev-0.1.0.orig/ffnetdev.h	2006-05-06 21:00:39.437264878 +0000
+++ vdr-ffnetdev-0.1.0/ffnetdev.h	2006-05-06 21:01:09.062255117 +0000
@@ -40,7 +40,7 @@
   virtual cOsdObject *MainMenuAction(void);
   virtual cMenuSetupPage *SetupMenu(void);
   virtual bool SetupParse(const char *Name, const char *Value);
-  virtual bool Active(void);
+  virtual cString Active(void);
   
   void SetPrimaryDevice();
   void RestorePrimaryDevice();
